src/main/jhipster/src/main/webapp/app/germplasm-details/pedigree/germplasm-neighborhood-tree.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="row form-group">
+	<div class="col-xs-12 col-md-12">
+		<div class="row">
+			<div class="col-12">
+				<form class="form-inline">
+					<div class="form-group">
+						<label>Number of Steps Backward</label>
+						<select [disabled]="isLoading" class="ml-1 form-control" name="numberOfStepsBackward" [(ngModel)]="numberOfStepsBackward" (change)="numberOfStepsChanged()">
```

Comment:
Implement `numberOfSteps` as constant in typescript, use angular to generate 1-`numberOfSteps` in html